static_frame/test/unit/test_frame.py

```diff
@@ -10438,32 +10440,33 @@ def test_frame_to_npz_a(self) -> None:
         with temp_file('.npz') as fp:
             f1.to_npz(fp)
             f2 = Frame.from_npz(fp)
-            f1.equals(f2, compare_dtype=True, compare_class=True, compare_name=True)
+            self.assertTrue(f1.equals(f2, compare_dtype=True, compare_class=True, compare_name=True))
 
     def test_frame_to_npz_b(self) -> None:
         f1 = ff.parse('s(10_000,2)|v(int,str)|c((I, ID),(str,dtD))|i(ID,dtD)').rename('foo')
 
         with temp_file('.npz') as fp:
             f1.to_npz(fp)
             f2 = Frame.from_npz(fp)
-            f1.equals(f2, compare_dtype=True, compare_class=True, compare_name=True)
+            self.assertTrue(f1.equals(f2, compare_dtype=True, compare_class=True, compare_name=True))
 
     def test_frame_to_npz_c(self) -> None:
         f1 = ff.parse('s(20,100)|v(int,str,bool)').rename('foo')
 
         with temp_file('.npz') as fp:
             f1.to_npz(fp)
             f2 = Frame.from_npz(fp)
-            f1.equals(f2, compare_dtype=True, compare_class=True, compare_name=True)
+            self.assertTrue(f1.equals(f2, compare_dtype=True, compare_class=True, compare_name=True))
 
+    @pytest.mark.skip("failing due to index name mismatch")
     def test_frame_to_npz_d(self) -> None:
         f1 = ff.parse('s(10,100)|v(int,str,bool,bool,float,float)').rename(
                 'foo', index='bar', columns='baz')
 
         with temp_file('.npz') as fp:
             f1.to_npz(fp)
             f2 = Frame.from_npz(fp)
-            f1.equals(f2, compare_dtype=True, compare_class=True, compare_name=True)
+            self.assertTrue(f1.equals(f2, compare_dtype=True, compare_class=True, compare_name=True))
```

Comment:
Looks like these tests were missing an `assert`. This test started failing due to a mismatch in index name. Just marked as skipped since I think this is out of scope.

NOTE: it looks like there are other places in this file that are missing `assertTrue`, at least in the to_npy tests.

Reply:
Thanks for isolating this!